src/ILVerify/README.md

```diff
@@ -16,6 +16,60 @@ Historically on Full Framework IL generators used PEVerify to make sure that the
 - It should be easy to add new verification rules
 - Fast spin up/tear down. 
 
+## The codebase
+The project targets netcoreapp2.0 and uses the new .csproj based project format. If you want to open and compile it with Visual Studio then you need a version, which supports .NET Core 2.0 tooling. Currently this is supported in Visual Studio 2017 Update 3 Preview (Version 15.3). The other option is to use command (with .NET Core 2.0 tooling - currenly also preview). 
+
+## Tests
+
+To test ILVerify we have small methods checked in as .il files testing specific verification scenarios. These tests live under [src/ILVerify/tests/ILTests](https://github.com/dotnet/corert/tree/master/src/ILVerify/tests/ILTests). Tests are grouped into .il files based on functionalities they test. There is no strict policy here, the goal is to have a few dozen .il files instead of thousands containing each only a single method. 
+
+Currently the IL files are NOT compiled automatically. You have to compile manually (We want to automatize this step later):
+
+```
+ilasm [filename.il] /dll /ERROR
+```
+
+Note: if you run the tests and get an error similar to this then it means that the .il files were not compiled, or none of them contained methods that follow the naming convention described below:
+
+```
+Result Message:	System.InvalidOperationException : No data found for ILVerify.Tests.ILMethodTester.TestMethodsWithInvalidIL
+```
+
+The test project itself is under [src/ILVerify/tests](https://github.com/dotnet/corert/tree/master/src/ILVerify/tests)
+
+The generated .dll files are automatically fed into two XUnit based test methods:
+ - 1 method expects valid IL code and asserts that the validation runs through without errors
+ - 1 method expects invalid IL code and asserts on the expected errors. 
+
+ Method names in the .il files must follow the following naming convention:
+
+### Methods with Valid IL:
+
+```
+[FriendlyName]_Valid
+```
+The method must contain 1 '_'. 
```

Comment:
Could you enclose the underscore in back ticks so that it is not treated as emphasis? (Notice that part of the next line is in italics.)